README.md

```diff
@@ -1,57 +1,250 @@
-Ballerina Twilio Connector
-===================
+# Ballerina Gmail Connector
```

Comment:
```suggestion
# Ballerina Twilio Connector
```

Reply:
Addressed in [e53338e](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/e53338ef5bd6bd3cc6640678b571a81183fbdbbf)